### ISSUE TITLE
Latency on Pi zero

### ISSUE THREAD

**Issue by sonium0** [FD-/RPiPlay#194]
Hi, first of all - awesome project.

I currently am seeing a latency of ~0.5 s on a Pi Zero in low latency mode. However in the readme it speaks about using OpenSSL. Is this already implemented or is this part of the setup one has to do manually?

**Comment by FD-** [comment]
Did you follow the tips for best performance listed in the readme?

**Comment by sonium0** [comment]
Yes, compiled with -O3, the other options are standard I understand. No other process then rpiplay -l is running. I use however I wireless connection, with the RPi configured as an accesspoint - so this should not be the issue.

**Comment by FD-** [comment]
Please check if a wired connection works better.
